cdap-api/src/main/java/io/cdap/cdap/internal/worker/api/RunnableTask.java

```diff
@@ -14,7 +14,7 @@
  * the License.
  */
 
-package io.cdap.cdap.common.internal.worker;
+package io.cdap.cdap.internal.worker.api;
```

Comment:
Don't move this to cdap-api. Move to cdap-system-app-api instead.

Reply:
Also, all `internal` package not meant to be used outside of the module itself.

Reply:
Is it ok to move to cdap-system-app-api considering other uses from app-fabric?

Reply:
Moved it to cdap-system-app-api. I see that aleardy app-fabric has dependency on it.